models/omen/gnosis/omen_gnosis_trades.sql

```diff
@@ -0,0 +1,123 @@
+{{ config(
+    schema = 'omen_gnosis',
+    alias = 'trades',
+    
+    partition_by = ['block_day'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_day', 'tx_hash', 'evt_index','outcomeSlot'],
+    post_hook='{{ expose_spells(blockchains  = \'["gnosis"]\',
+                                spell_type   = "project",
+                                spell_name   = "omen",
+                                contributors = \'["hdser"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2020-12-01' %}
+
+WITH
+
+trades AS (
+    SELECT
+        block_time
+        ,DATE_TRUNC('day', block_time) AS block_day
```

Comment:
i'd recommend explicitly casting to date, otherwise it'll default to `block_time` timestamp value